processing/src/main/java/io/druid/query/metadata/metadata/SegmentMetadataQuery.java

```diff
@@ -30,12 +31,16 @@
 import io.druid.query.spec.QuerySegmentSpec;
 import org.joda.time.Interval;
 
+import java.nio.ByteBuffer;
 import java.util.Arrays;
 import java.util.EnumSet;
+import java.util.List;
 import java.util.Map;
 
 public class SegmentMetadataQuery extends BaseQuery<SegmentAnalysis>
 {
+  public static final byte[] ANALYSIS_TYPES_CACHE_PREFIX = new byte[]{0x70};
```

Comment:
actually u already have typeBytes, so u can remove the analysis_types_cache_prefix entirely